test/proxy/run-plugin/runPlugin.tree

```diff
@@ -23,6 +23,5 @@ runPlugin.t.sol
             ├── when the plugin self-destructs
             │  └── it should return an empty response and send the ETH to the SELFDESTRUCT recipient
             └── when the plugin does not self-destruct
-               ├── it should run the plugin
-               └── it should emit a {RunPlugin} event
+               └── it should run the plugin and emit a {RunPlugin} event
```

Comment:
I preferred the tests to be separated

Reply:
I can understand why splitting tests would make sense in a test where lots of different things are happening.

But here, it doesn't seem like combining these 2 verifications is necessary and makes the tests a lot clearer. To me, the code duplication here feels more painful than the separation of concerns feels clearer.

In any case, I'm ok with whatever option you end up choosing. 🙌